apps/docs/app/analytics.tsx

```diff
@@ -63,7 +63,9 @@ function CookieConsent({
 		}
 	}
 
-	if (hasConsent !== 'unknown') return null
+	if (hasConsent === 'not-initialized' || hasConsent !== 'unknown') {
```

Comment:
does this extra logic of `not-initialized` get us anything here?
`'not-initialized' !== 'unknown'` so you can just leave this line as:
`if (hasConsent !== 'unknown') return null`

but then, why do we add `not-initialized`? :) seems like we might not need the extra complexity here

Reply:
True, we can drop the additional check.

We do need it tough. Go the https://tldraw.dev/, accept the cookies, then to a hard reload. You'll see the consent for a split second.